runelite-client/src/main/java/net/runelite/client/plugins/specialcounter/SpecialCounterConfig.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2018, Magic fTail
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.specialcounter;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+	keyName = "specialCounter",
+	name = "Special Attack Counter",
+	description = "Configuration for the special attack counter plugin"
+)
+public interface SpecialCounterConfig extends Config
+{
+	@ConfigItem(
+		keyName = "corpCounter",
+		name = "Corp Only",
+		description = "Show only special attac counter for Corperal Beast",
+		position = 1
+	)
+	default boolean corpCounter()
```

Comment:
Is there a special reason for it being corp only? I think if special attack counter should be disabled, it should be toggleable for all bosses rather than just one. Either by having a boolean for all supported bosses, or a new config option for maybe a multiple selection field.

Reply:
I agree, I would just limit it to bosses by default and not have a toggle for it.

Reply:
Making bosses toggleable do not makes much sense. Just disable plugin until you go bossing. That is why there is button for it.

Reply:
I would prefer if it only tracked corp since that is the only boss where this is useful, but discord was against that. I'm generally against forcing users into having to turn plugins off and on. If a plugin is only useful doing certain activities, it should only be active doing those activities.

Reply:
But, turning plugins on only when you need them is always best thing to do. No resources wasted. Also no, it is useful at Vorkath too. This is way too specific.